diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileChanges/last-build.bin b/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileChanges/last-build.bin differ
diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileHashes/fileHashes.lock b/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..eedee26
Binary files /dev/null and b/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/fileHashes/fileHashes.lock differ
diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/gc.properties b/node_modules/ffmpeg-kit-react-native/android/.gradle/6.8.1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/checksums/checksums.lock b/node_modules/ffmpeg-kit-react-native/android/.gradle/checksums/checksums.lock
new file mode 100644
index 0000000..0e99027
Binary files /dev/null and b/node_modules/ffmpeg-kit-react-native/android/.gradle/checksums/checksums.lock differ
diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/configuration-cache/gc.properties b/node_modules/ffmpeg-kit-react-native/android/.gradle/configuration-cache/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/ffmpeg-kit-react-native/android/.gradle/vcs-1/gc.properties b/node_modules/ffmpeg-kit-react-native/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/ffmpeg-kit-react-native/android/build.gradle b/node_modules/ffmpeg-kit-react-native/android/build.gradle
index e509ee4..f0921c7 100644
--- a/node_modules/ffmpeg-kit-react-native/android/build.gradle
+++ b/node_modules/ffmpeg-kit-react-native/android/build.gradle
@@ -25,12 +25,15 @@ def safeExtGet(String prop, String fallback) {
   rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
+def compileSdkVer = rootProject.ext.has('compileSdkVersion') ? rootProject.ext.get('compileSdkVersion') : 33
+def targetSdkVer = rootProject.ext.has('targetSdkVersion') ? rootProject.ext.get('targetSdkVersion') : 33
+
 android {
-  compileSdkVersion 30
+  compileSdkVersion compileSdkVer
 
   defaultConfig {
     minSdkVersion safeExtGet('ffmpegKitPackage', 'https').contains("-lts") ? 16 : 24
-    targetSdkVersion 30
+    targetSdkVersion targetSdkVer
     versionCode 451
     versionName "4.5.1"
   }
@@ -50,6 +53,7 @@ android {
 }
 
 repositories {
+  maven { url 'https://maven.aliyun.com/repository/public' }
   mavenCentral()
   jcenter()
   google()
